packages/paste-core/components/button/src/styles.ts

```diff
@@ -83,3 +83,52 @@ export const SizeStyles: {[key in ButtonSizes]: BoxStyleProps} = {
     lineHeight: 'lineHeight20',
   },
 };
+
+export const ToggleButtonStyles: BoxStyleProps = {
+  transition: 'background-color 150ms ease-in, box-shadow 150ms ease-in, color 150ms ease-in',
+  _pressed: {
+    backgroundColor: 'colorBackgroundPrimaryStronger',
+    color: 'colorTextWeakest',
+    boxShadow: 'none',
+  },
+  _pressed_hover: {
+    backgroundColor: 'colorBackgroundPrimary',
+    boxShadow: 'none',
+  },
+  _pressed_focus: {
+    boxShadow: 'shadowFocus',
+  },
+};
+
+export const ToggleIconButtonStyles: BoxStyleProps = {
+  transition:
+    'background-color 150ms ease-in, box-shadow 150ms ease-in, color 150ms ease-in, border-color 150ms ease-in',
+  borderWidth: 'borderWidth10',
+  borderStyle: 'solid',
+  borderColor: 'transparent',
+
+  _hover: {
+    borderColor: 'colorBorderPrimaryStronger',
+    backgroundColor: 'colorBackgroundPrimaryWeakest',
+  },
+  _focus: {
+    backgroundColor: 'colorBackgroundPrimaryWeakest',
+    boxShadow: 'shadowFocus',
+  },
+  _focus_hover: {
```

Comment:
thought: I was initially confused here by these hover, focus and focus_hover styles, since they are not for the `pressed` state, but are still spread onto the button. I realize now that this is because toggle styles are added when `pressed` is not undefined; when `pressed=false`, it's still defined, so these styles still apply. 

I'm back and forth on whether we mention `pressed=false` or not in the docs when we explain how to use it. There are props that we have people use in our system that we don't do this for. For example in button, `disabled` and `loading` are binary in that they are added and are true or aren't present and are false-y. `pressed` isn't like that since it's both telling our button that it's a toggle and which state of the toggle it's in. If we want a toggle button, you have to explicitly say that it's `pressed=true` or `pressed=false`. 

This might not be _that_ big of a mental leap for consumers, but maybe something to think about.

Reply:
Ohhhh true, I added more info about that on the docs site.